app/mindfulness/page.tsx

```diff
@@ -1,72 +1,11 @@
 "use client";
-import React, { useState, useEffect } from 'react';
 import Header from '../header';
 import Footer from '../footer';
-import Modal from 'react-modal';
-import { affirmations } from '../state';
-
-interface Button {
-  id: number;
-  text: string;
-}
 
 function Home() {
-  const [modalIsOpen, setModalIsOpen] = useState(false);
-  const [currentEditId, setCurrentEditId] = useState<number | null>(null);
-  const [editText, setEditText] = useState('');
-  const initialButtons: Button[] = [
-    { id: 1, text: 'Say what I want to say, whatever happens will help me grow' },
-    { id: 2, text: 'I can take up space' },
-    { id: 3, text: 'I have an important voice' },
-    { id: 4, text: 'Feel the tension and proceed' },
-    { id: 5, text: 'I have the right to stutter' },
-  ];
-
-  // Initialize state with saved data if it exists, otherwise use the initial data.
-  const [buttons, setButtons] = useState<Button[]>(() => {
-    const stringifiedAffirmations = affirmations.getAffirmationsAsString();
-    return stringifiedAffirmations ? JSON.parse(stringifiedAffirmations) : initialButtons;
-  });
-
-  useEffect(() => {
-    affirmations.setAffirmationsAsString(JSON.stringify(buttons));
-  }, [buttons]);
-
-  const openModal = (button: Button) => {
-    setModalIsOpen(true);
-    setCurrentEditId(button.id);
-    setEditText(button.text);
-  };
-
-  const closeModal = () => {
-    setModalIsOpen(false);
-  };
-
-  const handleEdit = () => {
-    setButtons((prevButtons) =>
-      prevButtons.map((button) =>
-        button.id === currentEditId ? { ...button, text: editText } : button
-      )
-    );
-    console.log(buttons);
-    closeModal();
-  };
 
-  const deleteButton = (id: number) => {
-    setButtons((prevButtons) => prevButtons.filter((button) => button.id !== id));
-  };
-
-  const addButton = () => {
-    const newId = Math.max(0, ...buttons.map((b) => b.id)) + 1; // Create a new ID
-    const newButton: Button = { id: newId, text: '' };
-    setButtons((prevButtons) => [...prevButtons, newButton]);
-    setModalIsOpen(true);
-    setCurrentEditId(newId);
-  };
-
-  const saveAffirmation = (text: string) => {
-    affirmations.setCurrentAffirmation(text);
-  };
+  const video1Path = 'affirmation_videos/1_Minute_Breathing_Exercise_Mini_Meditation_Simple_Breathing_Exercise_One_Minute_Breathe_Bubble.mp4'; // Replace with the actual path to your video1
```

Comment:
I think it's unlikely that we will host all the of affirmation videos within the app itself.  It seems more likely that we would embed videos from another host, like youtube/vimeo.

Reply:
@kexinF, can you upload them as unlist videos to youtube and use embedded link on the page? we may need to add youtube.com as trusted url in the  Zoom app config.